rebasehelper/helpers/process_helper.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import os
+import subprocess
+import tempfile
+
+import six
+
+from rebasehelper.constants import defenc
+from rebasehelper.logger import logger
+
+
+class ProcessHelper(object):
+
+    """Class for executing subprocess."""
+
+    DEV_NULL = os.devnull
+
+    @staticmethod
+    def run_subprocess(cmd, input_file=None, output_file=None):
+        """Runs the passed command as a subprocess.
+
+        Args:
+            cmd (list): Command with arguments to be run.
```

Comment:
I'd take an inspiration in actual `subprocess` module and call this `A sequence of program arguments.`.